impeller/renderer/command.cc

```diff
@@ -48,9 +48,10 @@ bool Command::BindResource(ShaderStage stage,
     case ShaderStage::kFragment:
       fragment_bindings.buffers[slot.binding] = {&metadata, view};
       return true;
+    case ShaderStage::kCompute:
+      FML_DCHECK(false);
```

Comment:
FML_UNREACHABLE for WIP stuff so we don't miss it?

Reply:
Done